tests/foreman/api/test_contentmanagement.py

```diff
@@ -1463,3 +1463,32 @@ def test_positive_sync_file_repo(
             sat_file = md5_by_url(f'{sat_repo_url}{file}')
             caps_file = md5_by_url(f'{caps_repo_url}{file}')
             assert sat_file == caps_file
+
+    @pytest.mark.tier3
+    def test_positive_allow_reregistration_when_dmi_uuid_changed(
+        self, module_org, rhel7_contenthost, default_sat
```

Comment:
Would it make sense to use the `rhel_contenthost` fixture instead of the `rhel7_contenthost` fixture?

You can limit RHEL versions to be spawned by these markers.
https://github.com/SatelliteQE/robottelo/blob/9f9163a646bae9318fa663a9748a1ac9734484c3/pytest_plugins/markers.py#L19-L20